Makefile

```diff
@@ -9,7 +9,7 @@ REGISTRY_AND_USERNAME := $(IMAGE_REGISTRY)/$(USERNAME)
 DOCKER_LOGIN_ENABLED ?= true
 
 ARTIFACTS := _out
-TOOLS ?= ghcr.io/talos-systems/tools:v0.3.0-21-g0026740
+TOOLS ?= ghcr.io/aleksi/tools:v0.3.0-23-g00e42a3
```

Comment:
Should be updated when https://github.com/talos-systems/tools/pull/128 is merged